### PR TITLE
To Do list: interactive list

### PR DESCRIPTION
**Pull request by aneallaryea100**
* Add a new JavaScript file and import it as a module:
* it will contain methods related to the status updates (completed: true / false).
* Add event listener to the checkbox (change).
* Update items object's value for completed key upon user actions.
* Implement a function for the "Clear all completed" button (use filter() method).
* Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.methods related to the status updates (completed: true / false).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
...
+- Webpack
+
+## Live Demo
+
+[Live Demo Link]()
+
+# Prerequisite
+* Modern Browser.Example: Firefox, chrome.
+* Windows 7 and above, MacOs, linux and android.
+# SET UP
```
Comment (elisha2kyakpo1) [inline README.md:21]: - [x] kindly add more steps here. Let's imagine that the person we are writing this is a non technical person. Highlight steps from cloning up to running the project on the user's local machine. `npm install` `git clone repo name`

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+});
+
+// edit input
+document.addEventListener('click', (event) => {
+  if (event.target.classList.contains('describetxt')) {
+    event.target.onchange = editElement;
+  }
+});
+
+// check checkbox
```
Comment (elisha2kyakpo1) [inline src/index.js]: - [x] Kindly remove all the commented code to make the codebase look clean

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @aneallaryea100, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The PR title and description looks professional 👍
- No linter errors 👍
- Descriptive commit messages 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
# Hi @aneallaryea100 👍🏽, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**